cmd/geth/retesteth.go

```diff
@@ -68,6 +68,7 @@ var (
 )
 
 type RetestethTestAPI interface {
+	SetRPCPayloadSize(ctx context.Context, size int) (int, error)
```

Comment:
it does not have to be rpc method. the limit could be set to like 10mb at the initialization of geth with retesteth flag. 
`geth retesteth`